config/routes.rb

```diff
@@ -136,6 +136,7 @@
         get "/:start_year", action: :show
       end
     end
+    resources :estimate_participants, only: %i[edit update], path: "estimate-participants"
```

Comment:
This should probably be nested inside `member` on `cohorts`, right?

Reply:
That would create something like `/schools/cohorts/:id/estimate-participants/:id/edit` ?